app/__init__.py

```diff
@@ -47,6 +49,16 @@
 def load_config(application):
     application.config['API_KEY'] = os.environ['TEMPLATE_PREVIEW_API_KEY']
     application.config['LOGOS'] = LOGOS
+    application.config['NOTIFY_ENVIRONMENT'] = os.environ['NOTIFY_ENVIRONMENT']
```

Comment:
It would be better to use the name and space_name under VCAP_APPLICATION, similar to here:-
https://github.com/alphagov/notifications-api/blob/master/app/cloudfoundry_config.py#L20